tests/robo/jenkins-robo-run.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
```

Comment:
This file is similar to `tests/nightly/jenkins-nightly-run.sh` except at the bottom, where we're running a different test suite.